CMakeLists.txt

```diff
@@ -75,6 +75,7 @@ option(RAJA_ENABLE_BOUNDS_CHECK "Enable bounds checking in RAJA::Views/Layouts"
 option(RAJA_TEST_EXHAUSTIVE "Build RAJA exhaustive tests" Off)
 option(RAJA_TEST_OPENMP_TARGET_SUBSET "Build subset of RAJA OpenMP target tests when it is enabled" On)
 option(RAJA_ENABLE_RUNTIME_PLUGINS "Enable support for loading plugins at runtime" Off)
+option(RAJA_ENABLE_DESUL_ATOMICS "Enable support of desul atomics" Off)
```

Comment:
@davidbeckingsale It appears that the desul tests, which rely on kokkos, are being included by default when ``-DDESUL_ENABLE_TESTS=Off`` is not specified. Would we be able to default this to "Off" similar to this?

Reply:
Yup, you can add `set(DESUL_ENABLE_TESTS Off CACHE BOOL "")` to turn them off here.